bluemira/equilibria/optimisation/harmonics/harmonics_constraint_functions.py

```diff
@@ -49,9 +49,9 @@ def f_constraint(self, vector: npt.NDArray) -> npt.NDArray:
         """Constraint function"""
         currents = self.scale * vector
 
-        result = self.a_mat[1:,] @ currents
+        result = self.a_mat[:,] @ currents
         return result - self.b_vec - self.value
 
     def df_constraint(self, _vector: npt.NDArray[np.float64]) -> npt.NDArray[np.float64]:
         """Constraint derivative"""
-        return self.scale * self.a_mat[1:,]
+        return self.scale * self.a_mat[:,]
```

Comment:
No need now, this will actually add an extra copy in here I think
```suggestion
        return self.scale * self.a_mat
```